containers/unit_tests/TestScatterView.hpp

```diff
@@ -707,8 +698,34 @@ TEST(TEST_CATEGORY, scatterview_devicetype) {
         10);
   }
 #endif
+#ifdef KOKKOS_ENABLE_HIP
+  if (std::is_same<TEST_EXECSPACE, Kokkos::Experimental::HIP>::value) {
+    using hip_device_type = Kokkos::Device<Kokkos::Experimental::HIP,
+                                           Kokkos::Experimental::HIPSpace>;
+    test_scatter_view<hip_device_type, Kokkos::Experimental::ScatterSum,
+                      double>(10);
+    test_scatter_view<hip_device_type, Kokkos::Experimental::ScatterSum,
+                      unsigned int>(10);
+    test_scatter_view<hip_device_type, Kokkos::Experimental::ScatterProd>(10);
+    test_scatter_view<hip_device_type, Kokkos::Experimental::ScatterMin>(10);
+    test_scatter_view<hip_device_type, Kokkos::Experimental::ScatterMax>(10);
+    using hip_pinned_device_type =
+        Kokkos::Device<Kokkos::Experimental::HIP,
+                       Kokkos::Experimental::HIPHostPinnedSpace>;
+    test_scatter_view<hip_pinned_device_type, Kokkos::Experimental::ScatterSum,
+                      double>(10);
+    test_scatter_view<hip_pinned_device_type, Kokkos::Experimental::ScatterSum,
+                      unsigned int>(10);
+    test_scatter_view<hip_pinned_device_type,
+                      Kokkos::Experimental::ScatterProd>(10);
+    test_scatter_view<hip_pinned_device_type, Kokkos::Experimental::ScatterMin>(
+        10);
+    test_scatter_view<hip_pinned_device_type, Kokkos::Experimental::ScatterMax>(
+        10);
+  }
+#endif
```

Comment:
Why is this part of this PR

Reply:
While removing `ROCm` with its tests I was checking if we were missing any tests in related files. Let me open a separate pull request with these changes.

Reply:
See #3336. Removed here.